### ISSUE TITLE
Some items are missing any dates, but we have `project_start`/`project_end` at our disposal

### ISSUE THREAD

**Issue by strogonoff** [relaton/relaton-3gpp#13]
Could we create items in `date` array for each of those dates? We could use literal string `project_start`/`project_end` as the `type` property, if no better alternative exists.

**Comment by ronaldtse** [comment]
As previously discussed, we should create the Release field for Relaton 3GPP, and also provide the Release date as a bibliographic date. I will file a separate issue for this and link.

**Comment by ronaldtse** [comment]
After #15 is implemented, we should apply the Release "end date" as the "release date" of 3GPP Specifications.
